priv/static/themes/ui-protection.css

```
@@ -0,0 +1,458 @@
+/* Warning flash messages */
+[data-phx-component*="flash"][class*="amber"],
+[data-phx-component*="flash"][class*="yellow"],
+.flash.warning,
+.alert.warning,
+div[phx-flash-kind="warning"],
+p[role="alert"][class*="amber"],
+p[role="alert"][class*="yellow"] {
+  background-color: #fffbeb !important; /* amber-50 */
+  color: #92400e !important; /* amber-800 */
+  border-color: #f59e0b !important; /* amber-500 */
+  border: 1px solid #f59e0b !important;
```

Biome (1.9.4)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)